externals_js/jquery-3.2.1.min.js

```diff
@@ -0,0 +1,4 @@
+/*! jQuery v3.2.1 | (c) JS Foundation and other contributors | jquery.org/license */
```

Comment:
tutaj raczej fajnie byłoby wrzucić to do folderu js znajdującego się w folderze externals, mogą dojść na przykład css zewnętrzne takie jak bootstrap